src/Servers/Kestrel/xunit.runner.json

```diff
@@ -3,5 +3,6 @@
   "appDomain": "denied",
   "methodDisplay": "method",
   "longRunningTestSeconds": 60,
-  "maxParallelThreads": -1
+  "maxParallelThreads": -1,
+  "diagnosticMessages": true
```

Comment:
I totally forgot about this! Yeah if you don't turn this on it doesn't actually *report* the long running tests... so annoying.

Reply:
I don't think this is actually being picked up by xunit 😢

Reply:
Also going to leave on diagnostic messages for now (to help us find more hangs if they occur). Will disable eventually 😄